src/services/common-util.service.ts

```diff
@@ -345,6 +347,12 @@ export class CommonUtilService implements OnDestroy {
         }
     }
 
+    getAppName(){
+        this.appVersion.getAppName().then((appName: any) => {
+            return appName;
+        });
+    }
+
```

Comment:
This method is not returning anything  . Make it return a  promise or an Observable.